modules/tinymce/src/core/test/ts/browser/focus/KeyboardShortcutMenuFocusTest.ts

```diff
@@ -0,0 +1,32 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyContentActions, TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+
+describe('browser.tinymce.core.focus.KeyboardShortcutMenuFocusTest', () => {
+
+  Arr.each([
+    { label: 'classic mode', options: {}},
+    { label: 'inline mode', options: { inline: true }},
+    { label: 'class bottom', options: { toolbar_location: 'bottom' }}
+  ], (tester) => {
+    context(tester.label, () => {
+      const hook = TinyHooks.bddSetup({
+        statusbar: false,
+        ...tester.options,
+        base_url: '/project/tinymce/js/tinymce'
+      }, [], true);
+
+      it('TINY-2884: Pressing Alt+F9 focuses the element path and escape from the toolbar will focus the editor', async () => {
+        const editor = hook.editor();
+        const doc = SugarDocument.getDocument();
+        setTimeout(() => {
+          TinyContentActions.keystroke(editor, 120, { alt: true });
+        }, 40);
```

Comment:
Why is this needed? If you ever need a setTimeout like this, then that's a pretty big red flag and it likely means there's some other issue.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/9b15ad55debfdf08d9ca4e128a48fda400c8d171